apps/console/src/features/applications/pages/application-edit.tsx

```diff
@@ -197,6 +200,25 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
         getApplication(id);
     }, []);
 
+    /**
+    * Fetch the identity provider id & name when calling the app edit through connected apps
+    */
+    useEffect(() => {
+        if(typeof history.location.state !== "object"){
+            return;
+        }
+
+        setisConnectedAppsRedirect(true);
+        type idpInfoType = {
```

Comment:
Can't we define this as an interface in ../models/application.ts file

Reply:
Added the interface and fixed in https://github.com/wso2/identity-apps/pull/3494/commits/3478ccc81887593d88f43d6c9659d6c0d0c10ed6